app/services/content-service/index.js

```diff
@@ -12,14 +12,15 @@
 
 import * as contentService from './server';
 import * as endpoints from '../../shared/constants/endpoints';
+import { logger } from './logging';
 
-process.on('uncaughtException', console.error);
-process.on('unhandledRejection', console.error);
+process.on('uncaughtException', (...args) => logger.error(...args));
+process.on('unhandledRejection', (...args) => logger.error(...args));
```

Comment:
Why these changes? Do we have to bind `logger` to `error` and other such functions?